src/components/Display.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
+  return (
+    <div id="display">
+      The result is:
+      { result }
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.number,
+};
```

Comment:
`result` should be a string [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png) ]